src/components/EvDialog.vue

```diff
@@ -0,0 +1,67 @@
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest -->
+
+<template>
+  <v-dialog :value="showDialog" @click:outside="onDeny" width="auto">
+    <v-card>
+      <v-card-title class="d-flex flex-row align-baseline">
+        <v-icon
+            size="large"
+            color="error"
+        >
+          mdi-alert-circle
+        </v-icon>
+        &nbsp;
+        {{ $props.title }}
+      </v-card-title>
+      <v-card-text>
+        <p>{{ $props.text }}</p>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn color="error" @click="onAccept">{{ $props.acceptText }}</v-btn>
+        <v-btn @click="onDeny">{{ $props.denyText }}</v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<style scoped lang="scss">
+
+</style>
+
+<script lang="ts">
+import Vue from "vue";
+export default Vue.extend({
+  props: {
+    showDialog: {
+      type: Boolean,
+      required: true,
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    text: {
+      type: String,
+      required: true,
+    },
+    acceptText: {
+      type: String,
+      required: true,
+    },
+    denyText: {
+      type: String,
+      required: true,
+    },
+  },
+  methods: {
+    onAccept() {
+      this.$emit("accept");
+    },
+    onDeny() {
+      this.$emit("deny", false);
```

Comment:
What's the purpose of the "false". It is not used as it seems

Reply:
This is the parameter for the function. In theory one could determine with that if the deny button was pressed or the user cliked in the background, but currently this was no meaning and can be removed.